docs/contents.rst

```diff
@@ -45,7 +45,6 @@ Table of Contents
    feature-store/feature-store
    concepts/runs-workflows
    serving/serving-graph
```

Comment:
Pls don't modify contents.rst without discussing. See https://github.com/mlrun/mlrun/pull/6253 for latest version